source/launchers/polychromatic.desktop

```diff
@@ -3,7 +3,9 @@ Type=Application
 Version=1.0
 Name=Polychromatic
 GenericName=Device Manager for RGB Lighting
-Comment=Configure connected lighting peripherals
+GenericName[fr]=Gestionnaire d'éclairage RGB des périphériques
+Comment=Configures connected lighting peripherals
```

Comment:
For `Comment`, the [Desktop Entry spec](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#recognized-keys) doesn't standardise plural vs non-plural. Considering other entries on the system and the specification's example, non-plural is the most common for English.

Please revert this line back to `Configure`.

Reply:
Oops ! I kept the wrong line when I rebased, sorry. It is fixed.